WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -260,10 +261,10 @@ public void onClick(View v) {
     }
 
     /*
-     * plans is a work-in-progress and is currently only exposed to alpha testers
+     * plans is a work-in-progress and isn't exposed yet
      */
     private static boolean isPlansEnabled() {
-        return AppPrefs.isInAppBillingAvailable();
+        return PackageUtils.isDebugBuild();
```

Comment:
IMO, that's a bit weird to remove the feature flag and enable the Plans in debug mode only. Can we keep the feature flag and eventually rename it to `PLANS_AVAILABLE`?